### ISSUE TITLE
Glint server crash

### ISSUE THREAD

**Issue by bakerac4** [typed-ember/glint#164]
Im finding that the server crashes when viewing an addon in node_modules.

![image](https://user-images.githubusercontent.com/9952936/115033958-feae8580-9e98-11eb-94de-8977520dfd40.png)

Usually just closing the files and restarting vscode fixes the issue

**Comment by dwilhelmi** [comment]
This happens to me also, but not just in node_modules, but when viewing *any* template that is not a match for my `include` pattern in my `.glintrc`. It errors with the following stack trace:

```
Error: Could not find source file: '/Users/dwilhelmi/code/mobile-web-client/app/components/r-cart/template.hbs'.
    at getValidSourceFile (/Users/dwilhelmi/code/mobile-web-client/node_modules/typescript/lib/typescript.js:149570:29)
    at Object.getSyntacticDiagnostics (/Users/dwilhelmi/code/mobile-web-client/node_modules/typescript/lib/typescript.js:149771:52)
    at GlintLanguageServer.getDiagnostics (/Users/dwilhelmi/code/mobile-web-client/node_modules/@glint/core/lib/language-server/glint-language-server.js:80:29)
    at Timeout._onTimeout (/Users/dwilhelmi/code/mobile-web-client/node_modules/@glint/core/lib/language-server/binding.js:69:52)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
```

I am using a restricted match in `.glintrc` to try to add this piecemeal rather than trying to update all my components to Glint. My `.glintrc` looks like this currently:
```
environment: ember-loose
include:
  - app/**/r-breadcrumbs/*
```

**Comment by dfreeman** [comment]
Thanks for reporting this! I thought we fixed this in #131, but this must be slightly different. I'll have a look 🙂
